docusaurus.config.js

```diff
@@ -5,100 +5,130 @@ const wallets = [
+  },
 ]
 
 const search = [
   {
     name: 'Infinity Search',
     image: 'infinity_search_logo.svg',
-    link: 'https://infinitysearch.co'
+    link: 'https://infinitysearch.co',
   },
   {
     name: 'Mojeek',
     image: 'mojeek_logo.svg',
-    link: 'https://www.mojeek.com'
-  }
+    link: 'https://www.mojeek.com',
+  },
 ]
 
 const tools = [
   {
-    name: 'Hugo + Webmonetization',
-    image: 'hugo_webmo_logo.svg',
-    link: 'https://github.com/sabinebertram/hugo-webmonetization-component'
+    name: 'Hugo Web Monetization Component',
+    image: 'logo-tool-hugo.svg',
+    link: 'https://github.com/sabinebertram/hugo-webmonetization-component',
+    desc: 'Theme component that enables web monetization on your entire Hugo website.',
   },
   {
-    name: 'Eleventy + Webmonetization',
-    image: '11ty_webmo_logo.png',
-    link: 'https://github.com/DanCanetti/eleventy-plugin-monetization'
+    name: 'Monetize 11ty',
+    image: 'logo-tool-11ty.svg',
+    link: 'https://github.com/DanCanetti/eleventy-plugin-monetization',
+    desc: 'An Eleventy plugin to monetize posts and site content by creating exclusive content areas.',
   },
   {
-    name: 'Gridsome + Webmonetization',
-    image: 'gridsome_webmo_logo.svg',
-    link: 'https://github.com/Sergix/gridsome-plugin-monetization'
+    name: 'gridsome-plugin-monetization',
+    image: 'logo-tool-gridsome.svg',
+    link: 'https://github.com/Sergix/gridsome-plugin-monetization',
+    desc: 'Web monetization plugin for Gridsome.',
   },
   {
-    name: 'Gatsby + Webmonetization',
-    image: 'gatsby_webmo_logo.svg',
-    link: 'https://github.com/mrmuhammadali/gatsby-plugin-monetization'
+    name: 'gatsby-plugin-monetization',
+    image: 'logo-tool-gatsby.svg',
+    link: 'https://github.com/mrmuhammadali/gatsby-plugin-monetization',
+    desc: 'A Gatsby plugin for react-monetize to speed up your integration with Web Monetization API.',
   },
   {
-    name: 'Angular + Webmonetization',
-    image: 'angular_webmo_logo.svg',
-    link: 'https://github.com/CDDelta/ngx-monetization'
+    name: 'ngx-monetization',
+    image: 'logo-tool-angular.svg',
+    link: 'https://github.com/CDDelta/ngx-monetization',
+    desc: 'Web Monetization API for Angular!',
   },
   {
-    name: 'VuePress + Webmonetization',
-    image: 'vuepress_webmo_logo.png',
-    link: 'https://github.com/spekulatius/vuepress-plugin-web-monetization'
+    name: 'Vuepress Plugin Web-Monetization',
+    image: 'logo-tool-vuepress.png',
+    link: 'https://github.com/spekulatius/vuepress-plugin-web-monetization',
+    desc: 'Adds a web-monetization meta tag to your VuePress website.',
   },
   {
-    name: 'Tessy + Webmonetization',
-    image: 'tessy_webmo_logo.svg',
-    link: 'https://github.com/VladimirMikulic/tessy'
+    name: 'Tessy',
+    image: 'logo-tool-tessy.svg',
+    link: 'https://github.com/VladimirMikulic/tessy',
+    desc: 'Advanced Web Monetization testing library with powerful declarative API.',
```

Comment:
This all looks much better, but it would be nice if we had consistent use of the term. 
Here we have "Web Monetization", "Webmonetization", "web-monetization" and "web monetization"
Seems like the rest of the page uses "Web Monetization"

Reply:
Should we take the PR and do this as a followup?